src/sentry/options/defaults.py

```diff
@@ -279,14 +279,30 @@
 )
 
 # API
-# Killswitch for apis to work with id or slug as path parameters
+# Killswitch for GA endpoints to work with id or slug as path parameters
 register(
     "api.id-or-slug-enabled",
-    default=False,
     type=Bool,
+    default=False,
     flags=FLAG_MODIFIABLE_BOOL | FLAG_AUTOMATOR_MODIFIABLE,
 )
 
+# Killswitch to enable EA endpoints to work with id or slug as path parameters
+register(
+    "api.id-or-slug-enabled-ea-endpoints",
+    type=Sequence,
+    default=[],
+    flags=FLAG_ALLOW_EMPTY | FLAG_AUTOMATOR_MODIFIABLE,
+)
+
```

Comment:
extreme nit 😶: could you group together the 3 API options to have no whitespace?
```suggestion
)
```